docs/units/equivalencies.rst

```diff
@@ -252,6 +252,26 @@ observations at high-energy, be it for solar or X-ray astronomy. Example::
     <Quantity 86.17332384960955 eV>
 
 
+Molar Mass AMU Equivalency
+--------------------------
+
+This equivalency allows conversion between the atomic mass unit and the equivalent g/mol.
+For reference to why this was added, refer to `NIST Mole Reference <http://physics.nist.gov/cuu/Units/mole.html>`__
+The following is an example of it's usage:
+
+    >>> import astropy.units as u
+    >>> import astropy.constants as const
+    >>> x = 1 * (u.g / u.mol) / const.N_A # doctest: +FLOAT_CMP
```

Comment:
I think `FLOAT_CMP` is only necessary for the Python command that outputs a value in the docstring to be compared.